qc/qc_adae.R

```diff
@@ -0,0 +1,20 @@
+###########################################################################
+#' developers : Thomas Neitmann/
+#' date: 16DEC2022
+#' QC ADSL
+#' modification History:
+#'  laxamanaj 08FEB2023 : updated to QC ADAE
+###########################################################################
+library(haven)
+library(diffdf)
+
+# -------------------------------------------#
+# QC/Check against original TDF ADAE dataset #
+# -------------------------------------------#
+adae <- read_xpt(file.path("submission", "datasets", "adae.xpt"))
+adae_orig <- read_xpt(file.path("adam", "adae.xpt"))
```

Comment:
adae_orig<- convert_blanks_to_na(adae_orig)

Reply:
@SHAESEN2 . are you suggesting to add this?

Reply:
added
```
> # QC/Check against original TDF ADAE dataset #
> # -------------------------------------------#
> adae <- read_xpt(file.path("submission", "datasets", "adae.xpt")) %>% 
+   convert_blanks_to_na()
> adae_orig <- read_xpt(file.path("adam", "adae.xpt")) %>% 
+   convert_blanks_to_na()
> 
> #---------#
> # Compare #
> #---------#
> diffdf(adae, adae_orig, keys = c("STUDYID", "USUBJID", "AESEQ"))
No issues were found!
```

Reply:
@SHAESEN2 or @thomas-neitmann If ok, please `Resolve conversation`.